torque-core/src/main/kotlin/com/workday/torque/Args.kt

```diff
@@ -25,20 +25,20 @@ data class Args(
         var testApkPaths: List<String> = emptyList(),
 
         @Parameter(
-                names = ["--annotations"],
+                names = ["--allowed-annotations"],
                 variableArity = true,
                 description = "Run only test methods WITH specified annotations. If multiple annotations are specified, will only run tests that are annotated with all of them. " +
                         "None by default and runs all tests. If used with other options, will run the intersection of them."
         )
-        var annotations: List<String> = emptyList(),
+        var allowedAnnotations: List<String> = emptyList(),
```

Comment:
just a thought, how about included/excluded?

Reply:
yeah that's better, I'll update